wlms/client.go

```diff
@@ -423,6 +423,69 @@ func (client *Client) Handle_CHAT(server *Server, pkg *packet.Packet) CmdError {
 	return nil
 }
 
+func (client *Client) Handle_CMD(server *Server, pkg *packet.Packet) CmdError {
+	var cmd, params string
+	if err := pkg.Unpack(&cmd, &params); err != nil {
+		return CmdPacketError{err.Error()}
+	}
+
+	if client.permissions != SUPERUSER {
+		return CmdPacketError{"DEFICIENT_PERMISSION"}
```

Comment:
deficient -> insufficient

Reply:
Hm, do you insist on this change? The string is not new but is already used some times in both the metaserver and Widelands. Changing it would mean modifying both components and adding an additional return-clause for legacy clients (those would still need the DEFICIENT-version).

Reply:
Let's leave it for now then.